packages/data-type/src/node/operations/action.ts

```diff
@@ -27,9 +30,41 @@ const GitHubActionCommandData = z.object({
 });
 export type GitHubActionCommandData = z.infer<typeof GitHubActionCommandData>;
 
+const WebSearchActionCommandUnconfiguredState = z.object({
+	status: z.literal("unconfigured"),
+});
+export type WebSearchActionCommandUnconfiguredState = z.infer<
+	typeof WebSearchActionCommandUnconfiguredState
+>;
+
+const WebSearchActionCommandConfiguredState = z.object({
+	status: z.literal("configured"),
+	commandId: z.custom<WebSearchActionCommandId>(),
+});
+export type WebSearchActionCommandConfiguredState = z.infer<
+	typeof WebSearchActionCommandConfiguredState
+>;
+
+const WebSearchActionCommandData = z.object({
+	provider: z.literal("web-search"),
+	state: z.discriminatedUnion("status", [
+		WebSearchActionCommandUnconfiguredState,
+		WebSearchActionCommandConfiguredState,
+	]),
+});
+export type WebSearchActionCommandData = z.infer<
+	typeof WebSearchActionCommandData
+>;
+
+const ActionCommandData = z.discriminatedUnion("provider", [
+	GitHubActionCommandData,
+	WebSearchActionCommandData,
+]);
+export type ActionCommandData = z.infer<typeof ActionCommandData>;
+
```

Comment:
This has been implemented, but it hasn't been connected to the UI, so it may not work when connected to the UI and may need to be changed.

Reply:
Thank you for your comment.
I will fix this during the feature implementation process. 🧑‍💻